packages/docusaurus-plugin-openapi/src/util.ts

```diff
@@ -6,10 +6,5 @@
  * ========================================================================== */
 
 export const isURL = (url: string) => {
-  try {
-    new URL(url);
-    return true;
-  } catch (e) {
-    return false;
-  }
+  return /^(?:[a-z]+:)?\/\//i.test(url);
```

Comment:
Could you write some test cases to cover this? Especially a test case for the urls/paths that were causing the issue